### PR TITLE
Release notes: add file

### PR DESCRIPTION
**Pull request by borys-belinsky**

### CODE REVIEW

File: source/release_notes/3.6.10-3.0.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  server startup, although a storage engine different from WiredTiger was used.
+- :jirabug:`PSMDB-257`: MongoDB could not be started with a group-readable ``key
+  file`` owned by ``root``.
+- :jirabug:`PSMDB-266`: In some cases, it was possible to add arbitrary
+  collections to the ``keydb`` directory which may only store ecryption data.
+
+Other bugs fixed: 
+:jirabug:`PSMDB-239`,
+:jirabug:`PSMDB-243`,
+:jirabug:`PSMDB-245`
```
Comment (tplavcic) [inline source/release_notes/3.6.10-3.0.rst]: PSMDB-245 should not be mentioned.

File: source/release_notes/3.6.10-3.0.rst [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+|PSMDB| is an enhanced, open source, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB 3.6 Community Edition.  It
+supports MongoDB 3.6 protocols and drivers.
+
+|PSMDB| extends Community Edition functionality by including the :ref:`inmemory`
+storage engine, as well as several enterprise-grade features.  Also, it includes
+:ref:`mongorocks` storage engine, which is now deprecated.  |PSMDB| requires no
+changes to MongoDB applications or code.
+
+
```
Comment (tplavcic) [inline source/release_notes/3.6.10-3.0.rst:49]: PSMDB-276 (Create tool to decrypt encrypted rollback files) is not mentioned. We have added a new tool to decrypt rollback files.
PSMDB-250 (nGram FTS implementation) is also not mentioned and I believe it's a community contribution so we should mention the name of the person who contributed this (check with @igorsol ).

**Review by tplavcic — CHANGES_REQUESTED** [review]

**Comment by tplavcic** [comment]
Release notes should mention that data at rest encryption is becoming GA in this version.

File: source/release_notes/3.6.10-3.0.rst [hunk 3]
```diff
@@ -0,0 +1,73 @@
+.. _3.6.10-3.0:
+
+================================================================================
+|PSMDB| |release|
+================================================================================
+
+:Date: October 31, 2018
```
Comment (igorsol) [inline source/release_notes/3.6.10-3.0.rst]: The date seems to be incorrect

File: source/release_notes/3.6.10-3.0.rst [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+|PSMDB| is an enhanced, open source, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB 3.6 Community Edition.  It
+supports MongoDB 3.6 protocols and drivers.
+
+|PSMDB| extends Community Edition functionality by including the :ref:`inmemory`
+storage engine, as well as several enterprise-grade features.  Also, it includes
+:ref:`mongorocks` storage engine, which is now deprecated.  |PSMDB| requires no
+changes to MongoDB applications or code.
+
+This release enables the encryption of rollback files. You can decrypt and
```
Comment (igorsol) [inline source/release_notes/3.6.10-3.0.rst]: It also enables encryption of temporary files used during external sorts.

**Review by tplavcic — APPROVED** [review]
This just needs info that data at rest encryption is becoming GA and then I think it looks ok.

**Review by igorsol — CHANGES_REQUESTED** [review]

File: source/release_notes/3.6.10-3.0.rst [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+fully-compatible, drop-in replacement for MongoDB 3.6 Community Edition.  It
+supports MongoDB 3.6 protocols and drivers.
+
+|PSMDB| extends Community Edition functionality by including the :ref:`inmemory`
+storage engine, as well as several enterprise-grade features.  Also, it includes
+:ref:`mongorocks` storage engine, which is now deprecated.  |PSMDB| requires no
+changes to MongoDB applications or code.
+
+Data at rest encryption becomes GA in |PSMDB| |release|. Also, this
+release enables the encryption of rollback files. You can decrypt and
```
Comment (igorsol) [inline source/release_notes/3.6.10-3.0.rst]: It also enables encryption of temporary files used during external sorts.

**Review by igorsol — CHANGES_REQUESTED** [review]

**Review by igorsol — APPROVED** [review]

**Pull request merged**